binds/python/bind_vasculature.cpp

```diff
@@ -12,9 +12,9 @@
 #include <morphio/vasc/properties.h>
 
 namespace py = pybind11;
-using namespace py::literals;
 
-void bind_vasculature(py::module &m) {
+static void bind_vasculature(py::module &m) {
+    using namespace py::literals;
```

Comment:
why the change?

Reply:
Because clang complains about using `using namespace` in a global scope.

Reply:
And the `static` because it complains about the fact there is no prototype otherwise

Reply:
Doesn't that `using namespace` trigger this: https://abseil.io/tips/153 ? Or am I reading it wrong?

Reply:
I guess it only brings in `_a`, so no worries

Reply:
or I can just bring _a with `using py::literals::operator""_a` ?

Reply:
(this syntax is sick by the way, I had no idea you could do that)